src/settings.rs

```diff
@@ -155,12 +155,14 @@ impl Default for Prompt {
 pub struct Behavior {
     #[serde(default = "def_bool_true")]
     pub validate_namespaces: bool,
+    pub print_context_in_exec: bool,
```

Comment:
This is a missing a default value in case it's not provided. It should be like `#[serde(default = "def_bool_false")]`

Reply:
Actually I misunderstood what this setting was doing. I thought that this setting forced the printing of headers even when the output is not a TTY.